frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -313,6 +313,16 @@ class Multiselect extends React.PureComponent<Props, State> {
                     Svg: {
                       style: {
                         color: theme.colors.darkGray,
+                        // Since the close icon is an SVG, and we can control its viewbox nor its attributes,
```

Comment:
nit: does this mean to say "can't control"?

Reply:
Yeah, thanks! Updated in [3c5444e](https://github.com/streamlit/streamlit/pull/6471/commits/3c5444e6f989dffcbede68aeb62c1fa0aa75246c)